_projects/example.md

```diff
@@ -0,0 +1,45 @@
+## Sample project
+
+```
```

Comment:
Consider removing from line 1 to line 3 
and line 45

Refer to any of our project file : 
https://raw.githubusercontent.com/coala/projects/master/_projects/aspects.md
and how it's rendered by github markdown
https://github.com/coala/projects/blob/master/_projects/aspects.md